packages/bpk-component-overlay/README.md

```diff
@@ -0,0 +1,51 @@
+# bpk-component-overlay
+
+> Backpack overlay component.
+
+## Installation
+
+```sh
+npm install bpk-component-overlay --save-dev
+```
+
+## Usage
+
+```js
+import React from 'react';
+import BpkText from 'bpk-component-text';
+import BpkOverlay, { BORDER_RADIUS_STYLES, OVERLAY_TYPES } from 'bpk-component-overlay';
+
+export default () => (
+  <div>
+    { /* Basic example */}
+    <BpkOverlay>
+      <BpkText>Hotels in Canada</BpkText>
+    </BpkOverlay>
+
+    { /* With the tint invisible */}
+    <BpkOverlay overlayType={OVERLAY_TYPES.none}>
+      <BpkText>Hotels in Canada</BpkText>
+    </BpkOverlay>
+
+    { /* With a border radius style */}
+    <BpkOverlay borderRadiusStyle={BORDER_RADIUS_STYLES.sm}>
+      <BpkText>Hotels in Canada</BpkText>
+    </BpkOverlay>
+
+    { /* With foreground content */}
+    <BpkOverlay foregroundContent={<BpkText>Visit Ottawa</BpkText>}>
+      <BpkText>Hotels in Canada</BpkText>
+    </BpkOverlay>
+  </div>
+);
+```
+
+## Props
+
+| Property  | PropType | Required | Default Value |
+| --------- | -------- | -------- | ------------- |
+| children | Node | true | - |
+| borderRadiusStyle | oneOf(BORDER_RADIUS_STYLES) | false | BORDER_RADIUS_STYLES.none |
```

Comment:
Is this how we document enums elsewhere? Would the format `sm | lg | xl`... be more useful?

Reply:
Normally I believe its 

```suggestion
| borderRadiusStyle | oneOf(BORDER_RADIUS_STYLES.none, BORDER_RADIUS_STYLES.sm) | false | BORDER_RADIUS_STYLES.none |
```

Reply:
I copied it from the image component which has the exact same prop, but yeah will change it!